lib/proxy.js

```diff
@@ -60,8 +69,28 @@ function startProxies(reqs, proc, command, emitter, portargs) {
     var localhost = 'localhost';
 
     var ports = command.proxy.split(',');
+
+    var ssl = {
+      cert: '',
+      key:  ''
+    };
+    if(command.ssl) {
+      var certDirectoryContents = fs.readdirSync(command.ssl);
+      certDirectoryContents.forEach(function(file) {
+        if (file.indexOf('.crt') >= 0) {
```

Comment:
Is this intended to check file extensions only? If so `/\.crt$/.test(file)` would probably be a better conditional.